components/layout/BlogLayout.js

```diff
@@ -22,12 +22,12 @@ export default function BlogLayout({ post, children }) {
 
   return (
     <BlogContext.Provider value={{ post }}>
-      <Container>
-        <NavBar />
-      </Container>
+      <div className="sticky top-0 w-full bg-white border-b border-gray-300 z-50">
+       <NavBar className="max-w-screen-xl block px-4 sm:px-6 lg:px-8 mx-auto" />
+      </div>
```

Comment:
I see that this code is used in other places. I suggest to create separate component for that like `StickuNavBar` or wrap current content inside `NavBar` component by that:

```
<div className="sticky top-0 w-full bg-white border-b border-gray-300 z-50">
```

node :)